docs/manifest-releaser.md

```diff
@@ -245,3 +247,31 @@ with a combined version of the library, i.e.,
 This functionality can be achieved by using the special `"."` path.
 `"."` indicates a release should be created when any changes are made to the
 codebase.
+
+## Plugins
+
+During a `manifest-pr` run, there is an opportunity to perform extra processing
+across all the source files and changelogs of all the configured packages.
+Unlike the individual releasers, which only have the context of the source files
+relevant to one package, a plugin receives all the current changes for all
+updated packages as well as the configuration containing every package. One
+place this is particularly useful is for monorepos that have intra-repo local
```

Comment:
I think it would be worth breaking this paragraph in half, the first half can talk about how the plugins work:

* what they're passed, and what they should output.

In this section I'd then have a code snippet that demonstrates the tiniest potential plugin that someone might right, e.g,. adding one additional update? _it can be a toy example._

The second paragraph can then describe why plugins would be useful, exactly as you have right now.

Reply:
let me know what you think of my `num-packages` plugin :-)